src/indent.cpp

```diff
@@ -3686,11 +3686,26 @@ void indent_text()
                  && options::indent_align_string())
          {
             log_rule_B("indent_align_string");
-            const int tmp = (xml_indent != 0) ? xml_indent : prev->column;
+            int indent;
 
+            if (xml_indent != 0)
+            {
+               indent = xml_indent;
+            }
+            else
+            {
+               Chunk *tmp = prev;
+
+               while (  tmp->prev
```

Comment:
you should use "GetPrev()" instead of "prev", which also takes away the need for the first check in the while loop.